source/ed/ed_persistor.cpp

```diff
@@ -251,11 +251,17 @@ void EdPersistor::insert_pois(const Georef &data) {
         if(itm.second.poi_type != nullptr){
             poi_type = std::to_string(itm.second.poi_type->id);
         }
-        this->lotus.insert({std::to_string(itm.second.id),
-                std::to_string(itm.second.weight),
-                this->to_geographic_point(itm.second.coord),
-                itm.second.name, "poi:" + itm.first, poi_type, std::to_string(itm.second.visible),
-                itm.second.address_number, itm.second.address_name});
+        this->lotus.insert({
+                std::to_string(itm.second.id), // id
+                std::to_string(itm.second.weight), // weight
+                this->to_geographic_point(itm.second.coord), // coord
+                itm.second.name, // name 
+                itm.first, // uri
```

Comment:
Poi imported with poi2ed won't be prefixed by `poi:` anymore.

Reply:
Now fixed in 6543acd. 
Do you know of any other dependence by any chance? I can't see any other. 
Also, do you know of any `poi.txt` file I can write tests against for `poi2ed` ?